metis-enrichment/metis-enrichment-rest/src/main/resources/log4j2.xml

```diff
@@ -9,5 +9,7 @@
 		<Root level="INFO">
 			<AppenderRef ref="Console"/>
 		</Root>
+		<!-- Set to DEBUG to log all requests to backend systems -->
+		<Logger name="eu.europeana.entity.client" level="DEBUG"/>
```

Comment:
need to discuss with Metis team, but I assume that log4j should be submitted only as template

Reply:
yes, but for now I will comment this. Usually, I use this for local testing.